Automattic/wp-calypso
# Start: add progress bar to new flows

## Proposed Changes

This adds the flow progress bar to the top of the `/start` flows. This allows us to unify the flow from `/start` to `/setup`. The progress bar will reflect the total amount of steps in the entire flow to give users an accurate estimate of where they are at.

I have preliminarily set a total step count to 9 roughly based on the progress bar position in the Figma. We may need to update this as the flow grows and is finalized.

<img width="1800" alt="Markup 2022-08-08 at 23 14 07" src="https://user-images.githubusercontent.com/33258733/183515828-aa884718-6491-4d6c-bf42-c9055089d23d.png">

<img width="1800" alt="Markup 2022-08-08 at 23 08 48" src="https://user-images.githubusercontent.com/33258733/183515689-4d6dfd06-5d13-4a5c-9f65-aa1738c4b008.png">

<img width="1800" alt="Markup 2022-08-08 at 13 36 35" src="https://user-images.githubusercontent.com/33258733/183515604-2adf1b33-cad4-4ad1-b7b1-b9636127cfa6.png">


## Testing Instructions

1. Pull branch and `yarn start`
2. Go to one of the new onboarding flows and verify the progress bar works
  - http://calypso.localhost:3000/start/newsletters/
  - http://calypso.localhost:3000/start/link-in-bio/
3. Got to the existing onboard flow and make sure there is no regression.
  - http://calypso.localhost:3000/start/users/

#### Pre-merge Checklist
